webpack.config.js

```diff
@@ -87,4 +88,10 @@ var config = {
   plugins: [new webpack.optimize.UglifyJsPlugin({mangle: false})]
 };
 
+// Support project overrides
+if (fs.existsSync('./project/webpack-project.config.js')) {
+  var projConfig = require('./project/webpack-project.config.js');
+  config = [config, projConfig];
```

Comment:
This doesn't work when I tested it with CAP's `project/webpack-project.config.js`
It does work with `config.push(projConfig);` instead

Reply:
![screenshot from 2017-10-24 15-59-52](https://user-images.githubusercontent.com/15268287/31965187-8471a93a-b8d4-11e7-82ff-d9c732e7a40a.png)

Reply:
weird -- works for me

Reply:
@ZainVirani it would work if you config is an array
the one currently on loris is not, are you sure urs is not modified?

Reply:
you're right -- disregard my comments